aspnetcore/mvc/advanced/custom-model-binding.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Custom Model Binding | Microsoft Docs
+author: ardalis
+description: Customizing model binding in ASP.NET Core MVC.
+keywords: ASP.NET Core, model binding, custom model binder
+ms.author: riande
+manager: wpickett
+ms.date: 4/10/2017
+ms.topic: article
+ms.assetid: ebd98159-a028-4a94-b06c-43981c79c6be
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: mvc/advanced/custom-model-binding
+---
```

Comment:
The model binder will work equally well with a viewmodel instead of individual action arguments:
replace viewmodel  with view model